rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc

```diff
@@ -61,6 +64,9 @@ Check the usage of a credential before changing or deleting it.
 . https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Create an application for Outlook.^]
 . https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth[Authenticate an EWS application by using OAuth.^]
 . https://learn.microsoft.com/en-us/exchange/client-developer/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth[Authenticate an IMAP, POP, or SMTP connection using OAuth.^]
+* To autofill the Google API service account credentials, obtain the credentials JSON file from your Google Cloud Console administrator. 
++
+If you have access to the Google Cloud console, follow the instructions in https://developers.google.com/workspace/guides/create-credentials#create_credentials_for_a_service_account[Create credentials for a service account^] to obtain credentials for your Google Cloud service account.
```

Comment:
Just a note that ^ is now added automatically for external links.

Reply:
Thank you for the reminder Tasha. Since there is no need to remove existing ^ I'll leave that here.